trigger/payloads_test.go

```diff
@@ -128,6 +129,31 @@ func (s *payloadSuite) TestSlack() {
 	s.Empty(m.Attachments)
 }
 
+func (s *payloadSuite) TestGetFailedTestsFromTemplate() {
+	test1 := task.TestResult{
+		URL:    "/test_log/failed",
+		Status: evergreen.TestFailedStatus,
+	}
+	test2 := task.TestResult{
+		URL:    "/test_log/success",
+		Status: evergreen.TestSucceededStatus,
+	}
+	t := task.Task{
+		Id:          "taskid",
+		DisplayName: "thetask",
+		Details: apimodels.TaskEndDetail{
+			TimedOut: false,
+		},
+		LocalTestResults: []task.TestResult{test1, test2},
+	}
+
+	tr, err := getFailedTestsFromTemplate(t)
+	s.NoError(err)
+	s.Require().Len(tr, 1)
+	s.Equal(tr[0].URL, "https://evergreen.mongodb.com/test_log/failed")
```

Comment:
where does this value come from? why is it true?

Reply:
It's just the URL from test1 with the base. If you wanted this test to be less magic-value I could change it to "https://evergreen.mongodb.com" + test1.URL, or get settings and use settings.Ui.Url + test1.URL, but that seems like overstepping part of what we want to test.